src/sdk/namespace_fs.js

```diff
@@ -969,24 +971,20 @@ class NamespaceFS {
     //    2.1 create .versions/ if it doesnt exist
     //    2.2 move old version (existing latest) to .versions/
     // 3. move new version to dest (key path)
-    async _move_to_dest_ver_enabled(fs_context, source_path, dest_path, { key }) {
-
+    async _move_to_dest_ver_enabled(fs_context, source_path, dest_path, target_file, { key }) {
         const cur_ver_info = await this._get_version_info(fs_context, dest_path);
```

Comment:
as we discussed it seems we need to retry on this whole part and not just in safe_move

Reply:
done